### PR TITLE
Collijk/feature/screen and vaccinate

### PR DESCRIPTION
**Pull request by collijk**
This PR adds a configuration parameter picked up from the specification that says some proportion of vaccines are guaranteed to be delivered to susceptible individuals.  This corresponds to a test and treat strategy vs. our existing blind vaccination strategy.

### CODE REVIEW

File: src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def __init__(self, *args, **kwargs):
 
     def _group_system(self, t: float, y: np.array, infectious: float,
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py:250]: Is there a docstring for these variables? Looking at this codebase for the first time, I'm confused.

File: src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py [hunk 2]
```diff
@@ -259,47 +259,36 @@ def _group_system(self, t: float, y: np.array, infectious: float,
...
-        # Effective vaccines are efficacious vaccines delivered to
-        # susceptible, unvaccinated individuals.
-        v_effective = s/n * v_efficacious
-        # Some effective vaccines confer immunity, others just protect
-        # from death after infection.
-        v_immune = p_immune * v_effective
-        v_protected = v_effective - v_immune
-
-        # vaccinated and unprotected come from all bins.
-        # Get count coming from S.
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py:271]: What is S?

File: src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py (hunk 1 above)
Comment (kathrynline) [reply src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py:250]: What is "screened" in general in this script?

**Review by kathrynline — APPROVED** [review]

**Comment by collijk** [comment]
this turned out to be an interesting experiment, but not something work keeping around.

**Pull request closed without merging**